spec/api-remote-spec.js

```diff
@@ -23,7 +23,29 @@ describe('remote module', () => {
 
   afterEach(() => closeWindow(w).then(() => { w = null }))
 
+  describe('remote.getGlobal', () => {
+    it('can return custom values', () => {
+      ipcRenderer.send('handle-next-remote-get-global', { test: 'Hello World!' })
+      assert.strictEqual(remote.getGlobal('test'), 'Hello World!')
+    })
+
+    it('throws when no returnValue set', () => {
+      ipcRenderer.send('handle-next-remote-get-global')
+      assert.throws(() => remote.getGlobal('process'), 'Invalid global: process')
+    })
+  })
+
   describe('remote.require', () => {
+    it('can return custom values', () => {
+      ipcRenderer.send('handle-next-remote-require', { test: 'Hello World!' })
+      assert.strictEqual(remote.require('test'), 'Hello World!')
+    })
+
+    it('throws when no returnValue set', () => {
+      ipcRenderer.send('handle-next-remote-require')
+      assert.throws(() => remote.require('electron'), 'Invalid module: electron')
```

Comment:
Why `expect`s have been replaced with `assert`s?
`chai` lib is available in the `4-0-x` branch, just add a `require` if it's not in the file yet.

Reply:
done